vars/main.yml

```diff
@@ -1,10 +1,13 @@
 ---
```

Comment:
You want to delete this file.  vars are essentially variables that are very hard to override.

Reply:
https://codereviewvideos.com/course/ansible-tutorial/video/variable-precedence-where-to-put-your-role-vars

Reply:
Swift, decisive action take in https://github.com/cisagov/ansible-role-nessus/pull/9/commits/7ed520a8082e97677e528c2ff6851934971d4942.